jwst/extract_1d/extract.py

```diff
@@ -3501,6 +3501,16 @@ def extract_one_slit(
     extract_model.log_extraction_parameters()
     extract_model.assign_polynomial_limits()
 
+    # store the extraction values we want to save in dictionary.
+    # define all the values to be None. If they are not valid for a mode
+    # they will not be written to fits header.
+    extraction_values = {}
+    extraction_values['xstart'] = None
+    extraction_values['xstop'] = None
+    extraction_values['ystart'] = None
+    extraction_values['ystop'] = None
+    extraction_values['width'] = None
```

Comment:
I don't see the "width" being used anywhere in subsequent code. Can it be removed from this dictionary?